eventhub/Module.md

```diff
@@ -1,55 +1,22 @@
+This module supports Event hub service operations like sending an event, sending batch events, 
 sending partition events and sending events with partition ID. It also supports Event hub management operations like 
 creating a new event hub, getting an event hub, updating an event hub, listing event hubs, deleting event hubs, 
 creating a new consumer group, getting consumer groups, listing consumer groups, listing partitions, getting partitions, 
 deleting consumer groups. The connector also provides the capability to handle publisher policy operations like getting 
-revoked publishers, revoking a publisher, and resume publishers.
-
-The connector will only be focusing on sending events to the event hub. The event hub connector will invoke the 
-REST APIs exposed via the Azure Event Hub. https://docs.microsoft.com/en-us/rest/api/eventhub/.
-
-The REST APIs fall into the following categories:
-* Azure Resource Manager: APIs that perform resource manager operations, and have /providers/Microsoft.EventHub/ as part 
-  of the request URI.
-* Event Hubs service: APIs that enable operations directly on the Event Hubs service, and have 
-  <namespace>.servicebus.windows.net/ in the request URI. The Event Hubs service API is focused on this implementation.
-
-# Compatibility
-|                     |    Version          |
-|:-------------------:|:-------------------:|
-| Ballerina Language  | Swan-Lake-Alpha5    |
-
-# Supported Operations
-
-## Azure Event Hubs Service Operations
-The `ballerinax/azure_eventhub` module contains operations related to accessing the Event Hubs service to perform 
-operations on event hubs. It includes operations to send event, send event with broker properties and user properties, 
-send event with partition key, send partition event, send batch events, send events with partition key, send batch 
-event with publisher ID. 
+revoked publishers, revoking a publisher, and resume publishers. This module will only be focusing on sending events to the event hub.
 
-## Azure Event Hubs Management Operations
-The `ballerinax/azure_eventhub` module contains operations related to accessing the Event Hubs service to performing 
-management operations on Event Hubs. It includes operations to create new event hub, get an event hub, 
-update an event hub, list event hubs, delete an event hub, create a new consumer group, get consumer group, 
-list consumer groups, list partitions, get partition, delete a consumer group. 
+This module supports Azure Event Hub [REST API 2014-01](https://docs.microsoft.com/en-us/rest/api/eventhub/) version.
 
-## Azure Event Hubs Publisher Policy Operations
-The `ballerinax/azure_eventhub` module contains operations related to performing publisher policy operations on 
-event hubs. It includes operations to revoke publisher, get revoked publishers, resume upblisher.
-
-# Prerequisites:
+## Configuring Connector
 
+### Prerequisites
```

Comment:
Other places say "Setting up prerequisites". Better decide and come to a common usage.

Reply:
Followed the common format for docs.